src/main/java/org/sciborgs1155/robot/vision/VisionConstants.java

```diff
@@ -0,0 +1,34 @@
+package org.sciborgs1155.robot.vision;
+
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.apriltag.AprilTagFields;
+import edu.wpi.first.math.Matrix;
+import edu.wpi.first.math.VecBuilder;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Transform3d;
+import edu.wpi.first.math.geometry.Translation3d;
+import edu.wpi.first.math.numbers.N1;
+import edu.wpi.first.math.numbers.N3;
+
+public class VisionConstants {
+  public static record CameraConfig(String name, Transform3d robotToCam) {}
+
+  public static final AprilTagFieldLayout TAG_LAYOUT = null;
+
+  public static final CameraConfig FRONT_CAMERA_CONFIG =
+      new CameraConfig(
+          "placeholder",
+          new Transform3d(new Translation3d(0.5, 0.0, 0.5), new Rotation3d(0, 0, 0)));
+  public static final CameraConfig SIDE_CAMERA_CONFIG =
+      new CameraConfig(
+          "placeholder",
+          new Transform3d(new Translation3d(0.5, 0.0, 0.5), new Rotation3d(0, 0, 0)));
+
+  // The layout of the AprilTags on the field
+  public static final AprilTagFieldLayout kTagLayout =
```

Comment:
this is the same variable as `TAG_LAYOUT`, it should be the thing you pass into the sim vision system as well